src/userscript-polyfill.js

```diff
@@ -0,0 +1,34 @@
+globalThis.IS_USERSCRIPT = true;
+
+globalThis._localStorage_browser_polyfill = {
```

Comment:
`_browser_userscript_polyfill` and `_localStorage_browser_polyfill` have inconsistent names (camel case mixed in, different order).